### PR TITLE
WT-8158 Create an infrastructure to autoload gdb scripts along with a sample script

### PR DESCRIPTION
**Pull request by agorrod**
Add some GDB macros to help debugging WiredTiger

### CODE REVIEW

**Review by sulabhM — APPROVED** [review]

**Comment by agorrod** [comment]
@sulabhM  - I've gotten this as far as I think I'm going to as a starting point. Would you mind taking a look?

File: tools/gdb_commands/wt_debug_script_update.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    wt_block_header_size = 12
+    page_bytes = gdb.selected_inferior().read_memory(int(dsk.address) + wt_page_header_size + wt_block_header_size, int(dsk['mem_size'])).tobytes()
+    print("Dsk:\n" + str(page_bytes))
+
+def dump_handle(dhandle):
+    print("Dumping: " + dhandle['name'].string())
+    btree = get_btree_handle(dhandle)
+    root = btree['root']
+    root_page = root['page'].dereference()
+    #dbg('btree', get_btree_handle(user))
```
Comment (quchenhao) [inline tools/gdb_commands/wt_debug_script_update.py:141]: Can we pass some arguments to enable these debug prints?

Comment (agorrod) [reply tools/gdb_commands/wt_debug_script_update.py]: I thought about this and just turned them on in the end. It's easier for someone to disable something that's too noisy than to remember to enable it.

I think this is very much a starting point. I'd like to improve both how these are loaded and the consistency of naming/interface. Hopefully whats here now will be a reasonable starting point for that.

**Comment by agorrod** [comment]
@quchenhao would you mind taking another look?

**Review by quchenhao — APPROVED** [review]
lgtm

**Pull request merged**